backend/src/main/java/org/polsl/backend/entity/Affiliation.java

```diff
@@ -37,6 +37,10 @@
   public Affiliation() {
   }
 
+  public String getFullName() {
+    return String.format("%s %s %s", firstName, lastName, location);
+  }
+
```

Comment:
Nie potrzebujemy tego tutaj - istnieje już metoda `generateName` w klasie `AffiliationService`, która robi praktycznie to samo, tylko w nieco inny sposób. Powinniśmy zawsze korzystać z jednej opcji